mava/wrappers/jaxmarl.py

```diff
@@ -79,14 +79,14 @@ def unbatchify(x: Array, agents: List[str]) -> Dict[str, Array]:
 
 
 def merge_space(
-    spec: Dict[str, Union[jaxmarl_spaces.Box, jaxmarl_spaces.Discrete]]
+    spec: Dict[str, Union[jaxmarl_spaces.Box, jaxmarl_spaces.Discrete]],
 ) -> jaxmarl_spaces.Space:
     """Convert a dictionary of spaces into a single space with a num_agents size first dimension.
 
     JaxMarl uses a dictionary of specs, one per agent. For now we want this to be a single spec.
     """
     n_agents = len(spec)
-    single_spec = copy.deepcopy(list(spec.values())[0])
+    single_spec = copy.deepcopy(next(iter(spec.values())))
```

Comment:
Is this a change that ruff made? The initial solution looks a bit better to me.

Reply:
Ye this is from ruff, I actually prefer it now, but I don't think it's a big deal